src/hyp3_isce2/insar_stripmap.py

```diff
@@ -4,13 +4,21 @@
 
 import argparse
 import logging
-import os
 import site
 import sys
 from pathlib import Path
 
+from hyp3_isce2 import stripmapapp_alos as stripmapapp
+
 from hyp3lib.aws import upload_file_to_s3
 from hyp3lib.image import create_thumbnail
+from hyp3_isce2.dem import download_dem_for_isce2
+import zipfile
+import glob
+import os
+
+import asf_search as asf
```

Comment:
This is how we typically import this module, so we'll want to do the same here for consistency.
```suggestion
import asf_search
```